docs/features/security/auth/jwt.md

```diff
@@ -62,4 +62,55 @@ public class Startup
         });
     }
 }
+```
+
+### Custom Claim validation
+
+It allows validating not only on the audience claim in the JWT token, but any type of custom claim that needs to be verified
+
+```csharp
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
```

Comment:
Based on https://github.com/arcus-azure/arcus.webapi/pull/179/files#diff-48f283b1ac9786c91d89d9bb04fcc05cf8d48411eb1086840a67a7de9dcf98cfR33 this looks outdated, is that correct?

Reply:
this is the latest version, nothing has changed here